ftw/calendarexport/tests/test_export.py

```diff
@@ -34,9 +40,39 @@ def test_export(self):
         view()
         feeddata = view.feeddata()
         feed_list = feeddata.split('BEGIN:VEVENT')
-        self.assertIn('SUMMARY:event1\nDTSTART:20140123T100000Z\n'
-        'DTEND:20140123T120000Z\nCLASS:PUBLIC\nEND:VEVENT\n', feed_list[1])
-        self.assertIn('SUMMARY:event2\nDTSTART:20140125T100000Z\n'
-        'DTEND:20140125T120000Z\nCLASS:PUBLIC\nEND:VEVENT\nEND:VCALENDAR\n',
+
+        self.assertIn(
+            'SUMMARY:event1\nDTSTART:20140123T100000Z\n'
+            'DTEND:20140123T120000Z\nCLASS:PUBLIC\nEND:VEVENT\n',
+            feed_list[1])
+        self.assertIn(
+            'SUMMARY:event2\nDTSTART:20140125T100000Z\n'
+            'DTEND:20140125T120000Z\nCLASS:PUBLIC\nEND:VEVENT\nEND:VCALENDAR\n',
             feed_list[2])
-        
+
+    def test_dx_export(self):
+        folder = create(Builder('event folder'))
+        today = datetime.now()
+        tomorrow = today + timedelta(days=1)
+        dayaftertomorrow = tomorrow + timedelta(days=1)
+        event1 = create(Builder('event page')
+                        .titled(u'Event One')
+                        .within(folder)
+                        .starting(today)
+                        .ending(tomorrow))
+        event2 = create(Builder('event page')
+                        .titled(u'Event Two')
+                        .within(folder)
+                        .starting(tomorrow)
+                        .ending(dayaftertomorrow))
+
+        view = folder.restrictedTraverse('export_ics')
+        uids = list()
+        uids.append(event1.UID())
+        uids.append(event2.UID())
+        self.portal.REQUEST.form['uids'] = uids
+        view()
+        feeddata = view.feeddata()
+
+        self.assertIn('SUMMARY:Event One', feeddata)
```

Comment:
Would it be helpful to assert the complete feed data, comparing it with an asset?
This test currently only tests that there is a summary. If, for example, the dates are missing or wrong the test would not fail.

When asserting the complete feed, the [freezing the date](https://github.com/4teamwork/ftw.testing#freezing-datetime-now) will be helpful.

Just an idea 😉